ple/games/donkeykong/Game.py

```diff
@@ -0,0 +1,197 @@
+__author__ = 'Batchu Vishal'
+import pygame
+import sys
+from pygame.locals import *
+from Board import Board
+from .. import base
+
+'''
+This class defines the logic of the game and how player input is taken etc
+We run one instance of this class at the start of the game, and this instance manages the game for us.
+'''
+
+
+class DonkeyKong(base.Game):
+    def __init__(self):
+        '''
+        Set the height and width for the game element
+        FPS is set to 30 frames per second
+        '''
+        self.height = 520
+        self.width = 1200
+
+        actions = {
+                "left": K_a,
+                "right": K_d,
+                "jump": K_SPACE,
+                "up": K_w,
+                "down": K_s
+        }
+        base.Game.__init__(self, self.width, self.height, actions=actions)
+        self.init()
+
```

Comment:
I just realized on my end I should allow base.Game to accept rewards as a kwarg. For now can you put a dict of rewards in? You can put the values in you used in the game. Eg.

``` python
self.rewards = {
"positive": 5.0, #coin pickup?
"negative": -25.0,
"tick": 0.0,
"loss": -50.0,
"win": 50.0
```

You can pick the values which you feel are accurate for each case. For this game just keep tick reward to 0.0